helios-tools/src/main/java/com/spotify/helios/cli/command/HostListCommand.java

```diff
@@ -109,25 +104,36 @@ public String apply(final HostStatus.Status input) {
         .choices(statusChoices.toArray(new String[statusChoices.size()]))
         .help("Filter hosts by its status. Valid statuses are: " + statusChoicesString);
 
-    hostSelectorsArg = parser.addArgument("-l", "--labels")
-        .action(append())
+    hostSelectorsArg = parser.addArgument("-s", "--selector")
         .setDefault(new ArrayList<String>())
         .nargs("+")
-        .help("Host selector expression. Hosts matching this expression will be part of the " +
-              "deployment-group. Multiple conditions can be specified, separated by spaces (as " +
-              "separate arguments). If multiple conditions are given, all must be fulfilled. " +
-              "Operators supported are =, !=, in and notin. Example: foo=bar baz!=qux");
+        .help("Host selector expression. The list of hosts will be filtered to match only those "
+              + "whose labels match all of the supplied expressions. "
+              + "Multiple expressions can be specified, separated by spaces "
+              + "(e.g. `-s site=foo bar!=yes`). " +
```

Comment:
That's probably because you removed `action(append())`. With the current implementation specifying `-l` seems to work. OTOH only specifying `-l` once also seems to work.

I just wonder how this interacts with positional arguments...